parlai/tasks/coco_caption/agents.py

```diff
@@ -11,10 +11,8 @@
 from .build_2017 import build as build_2017
 from .build_2017 import buildImage as buildImage_2017
 
-try:
-    import torch  # noqa: F401
-except ImportError:
-    raise ImportError('Need to install Pytorch: go to pytorch.org')
+
+import torch  # noqa: F401
```

Comment:
I think you can drop anything leftover with a `# noqa: F401`. They were imports only for checking versioning.